docs/docs/api/endpoints/messages.md

```diff
@@ -67,3 +66,53 @@ import MessagesSend from './messages-send.mdx'
 import GoogleMessagesSend from './google-messages-send.mdx'
 
 <GoogleMessagesSend />
+
+## Suggest replies
+
+`POST /messages.suggestReplies`
+
+Suggest a set of replies for a given message id. UI clients can show these to agents to make responding to user inquiries
+faster and more efficiently.
+
+**Sample request**
+
+```json5
+{
+  "message_id": "uuid",
+  "suggestions": {
+    "suggestion-id-1": {
+      "content": {"text": "Great that it worked. Is there anything else you need?"}
+    },
+    "suggestion-id-2": {
+      "content": {"text": "Have a nice day!"}
```

Comment:
unrelated to the PR but want to make a mental note of this for us by sharing first thoughts. I guess we have to explain somewhere in the docs how "content works". The way I see it the best place for this is the sources introduction (but that's also a different conversation). the point is that we need a place we can refer to in the docs where we encounter "content" in apis

Reply:
[Agreed](https://github.com/airyhq/airy/issues/1075)